README.md

```diff
@@ -25,21 +25,33 @@ $ ./chimera -c <yaml_config_file> -o <output_path> my_cpp_header1.h my_cpp_heade
 - llvm 3.6 or above (+ tools)
 - libedit
 - yaml-cpp
+- boost
 
 **On Ubuntu**
 
 ```bash
-sudo add-apt-repository 'deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.6 main'
-wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
-sudo apt-get update
-sudo apt-get install llvm-3.6-dev llvm-3.6-tools libclang-3.6-dev libedit-dev libyaml-cpp-dev
+$ sudo add-apt-repository 'deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty-3.6 main'
+$ wget -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
+$ sudo apt-get update
+$ sudo apt-get install llvm-3.6-dev llvm-3.6-tools libclang-3.6-dev libedit-dev libyaml-cpp-dev libboost-dev
+$ git clone https://github.com/personalrobotics/chimera.git
+$ cd chimera
+$ mkdir build && cd build
+$ cmake ..
+$ make
 ```
 
 **On Mac OS X**
 
 ```bash
-brew install yaml-cpp --with-static-lib
-PKG_CONFIG_PATH=/usr/local/Cellar/yaml-cpp/0.5.2/lib/pkgconfig cmake -DLLVM_DIR=/usr/local/opt/llvm/share/llvm/cmake ..
+$ brew install boost llvm
+$ brew install yaml-cpp --with-static-lib 
+$ git clone https://github.com/personalrobotics/chimera.git
+$ cd chimera
+$ mkdir build && cd build
+$ PKG_CONFIG_PATH=/usr/local/Cellar/yaml-cpp/0.5.2/lib/pkgconfig cmake -DCMAKE_BUILD_TYPE=Release \
+    -DLLVM_DIR=/usr/local/Cellar/llvm/3.9.1/lib/cmake/llvm/ ..
```

Comment:
I did some poking around in Homebrew and managed to find a way of generating the correct path:
```bash
-DLLVM_DIR=$(brew --prefix llvm)/lib/cmake/llvm
```
I much prefer this to hardcoding a version number, since it will work if the user copies and pastes it.

Also, I seem to have `yaml-cpp` in my default `PKG_CONFIG_PATH`. If the same is true for you, I'd like to remove the explicit definition of `PKG_CONFIG_PATH`. If it is necessary, please replace the hardcoded path with:
```
PKG_CONFIG_PATH=$(brew --prefix yaml-cpp)/lib/pkgconfig
```